src/pages/Admin/Venue/AnnouncementOptions/AnnouncementOptions.scss

```diff
@@ -0,0 +1,19 @@
+@import "scss/constants";
+
+.AnnouncementOptions {
+  display: flex;
+  justify-content: space-between;
+  padding-top: $spacing--lg;
+
+  &__left-side {
+    font-size: $font-size--lg;
+    margin-top: $spacing--xs;
+  }
+
+  &__edit-button {
+    height: $submit-button-diameter;
+    width: $announce-button-width;
+    background-color: $white;
+    color: $primary--live;
+  }
```

Comment:
can you revise if you need all these styles now when you applied `ButtonNG`. You can use `variant="primary"` for the color, for sure.

Reply:
fixed with variants d93f7db